lib/Common/ConfigFlagsList.h

```diff
@@ -1477,6 +1487,9 @@ FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup i
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
 
 FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
+FLAGR (Boolean, WriteBarrierTest, "Always return true while checking barrier to test recycler regardless of annotation", DEFAULT_CONFIG_WriteBarrierTest)
+FLAGR (Boolean, EnableHardwareWriteBarrier, "Use to turn off write watch to test software write barrier on windows", DEFAULT_CONFIG_EnableHardwareWriteBarrier)
+FLAGR (Boolean, EnableBGFreeZero, "Use to turn off background freeing and zeroing to simulate linux", DEFAULT_CONFIG_EnableBGFreeZero)
```

Comment:
Should these flags be added to retail? Looks they should stay in debug/test builds?